server/liveblog/syndication/producer.py

```diff
@@ -213,17 +245,28 @@ def _delete_producer_blogs_syndicate(producer_id, blog_id, consumer_blog_id):
             return api_error('Unable to unsyndicate producer blog.', response.status_code)
 
 
-@producers_blueprint.route('/api/producers/<producer_id>/syndicate/<blog_id>', methods=['POST', 'DELETE'])
+@producers_blueprint.route('/api/producers/<producer_id>/syndicate/<blog_id>', methods=['POST', 'PATCH', 'DELETE'])
 def producer_blogs_syndicate(producer_id, blog_id):
-    consumer_blog_id = request.get_json().get('consumer_blog_id')
-    auto_publish = request.get_json().get('auto_publish')
+    data = request.get_json()
+    consumer_blog_id = data.get('consumer_blog_id')
+    auto_publish = data.get('auto_publish')
+    start_date = data.get('start_date')
+
     if not consumer_blog_id:
         return api_error('Missing "consumer_blog_id" in form data.', 422)
 
+    if start_date:
+        try:
+            start_date = str_to_date(start_date)
+        except ValueError:
+            return api_error('start_date is not valid.', 400)
+
     if request.method == 'DELETE':
         return _delete_producer_blogs_syndicate(producer_id, blog_id, consumer_blog_id)
+    elif request.method == 'PATCH':
+        return _update_producer_blogs_syndicate(producer_id, blog_id, consumer_blog_id, start_date)
```

Comment:
auto_publish can be updated by sending PATCH request to the blueprint url. This is not actually implemented by the javascript client, but it's available.